data/events/scripts/monster.lua

```diff
@@ -42,7 +42,21 @@ function Monster:onDropLoot(corpse)
 			end
 		end
 
+		-- Vip system
+		local vipPercentLoot = 0
+		if (configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED)
+			and configManager.getBoolean(configKeys.VIP_SYSTEM_LOOT_ENABLED)
+			and player and player:isVip()) then
+			vipPercentLoot = (configManager.getNumber(configKeys.VIP_SYSTEM_LOOT_PERCENT) / 100)
+		end
+
 		for i = 1, #monsterLoot do
+			if (configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED)
+				and configManager.getBoolean(configKeys.VIP_SYSTEM_LOOT_ENABLED)
+				and vipPercentLoot > 0) then
+				monsterLoot[i].chance = monsterLoot[i].chance + (monsterLoot[i].chance * vipPercentLoot)
```

Comment:
Nesse caso nao e necessario checar se vipPercentLoot e maior que 0. Se vipPercentLoot for 0, a conta vai ser monsterLoot[i].chance + 0.